barretenberg/cpp/src/barretenberg/dsl/acir_proofs/honk_contract.hpp

```diff
@@ -254,6 +254,18 @@ library Honk {
         G1Point lagrangeLast;
     }
```

Comment:
This pattern of having to copy updates from Solidity to here is really not great (although same thing was done for Plonk). So we should come up with an alternative